### PR TITLE
Survey results

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Added section to display results os previous surveys.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-56693

### CODE REVIEW

File: webapp/src/components/common/button/button.tsx
Comment (larkox) [inline webapp/src/components/common/button/button.tsx:1]: :heart:

**Review by larkox — COMMENTED** [review]
LGTM

File: webapp/package-lock.json [hunk 1]
```diff
@@ -5851,7 +5851,8 @@
     },
     "node_modules/classnames": {
       "version": "2.5.1",
-      "license": "MIT"
+      "resolved": "https://registry.npmjs.org/classnames/-/classnames-2.5.1.tgz",
+      "integrity": "sha512-saHYOzhIQs6wy2sVxTM6bUDsQO4F50V9RQ22qBpEdCW+I+/Wmke2HOl6lS6dTpdxVhb88/I6+Hs+438c3lfUow=="
```
Comment (M-ZubairAhmed) [inline webapp/package-lock.json:5855]: what changed this

**Review by M-ZubairAhmed — APPROVED** [review]

File: webapp/src/components/common/button/button.tsx
Comment (harshilsharma63) [reply webapp/src/components/common/button/button.tsx:1]: 👍

File: webapp/src/components/systemConsole/surveyResults/surveyResults.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                <div className='horizontal resultHeader'>
+                    <div className='startDate'>{'Start date (UTC)'}</div>
+                    <div className='endDate'>{'End date (UTC)'}</div>
+                    <div className='npsScore'>{'NPS Score'}</div>
+                    <div className='receiptsCount'>{'Sent to'}</div>
+                    <div className='responseCount'>{'Responses'}</div>
+                    <div className='actions'>{''}</div>
+                </div>
+                <div className='resultBody'>
+                    {renderedRows}
```
[the remark this answers is not in the document]
Comment (harshilsharma63) [reply webapp/src/components/systemConsole/surveyResults/surveyResults.tsx:137]: Thats a great point! Added a basic view for now.

@abhijit-singh LMK if we need to make any changes here. I added this just as a very basic version.

Screenshot 2024-04-08 at 12 43 48 PM

File: webapp/package-lock.json (hunk 1 above)
Comment (harshilsharma63) [reply webapp/package-lock.json:5855]: I might have used incorrect npm version previously and this fix might have been caused by using the right version again. But not sure though.

**Comment by harshilsharma63** [comment]
@larkox I've added the empty view now. UI can be updated whenever later.

**Comment by harshilsharma63** [comment]
Will merge once #6 is approved.

**Pull request merged**